testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTableStatistics.java

```diff
@@ -1467,6 +1467,30 @@ public Object[][] testComputeFloatingPointStatisticsDataProvider()
         };
     }
 
+    @Test
+    public void testComputeStatisticsForTableWithOnlyDateColumns()
+    {
+        String tableName = "test_compute_statistics_with_only_date_columns";
+        onTrino().executeQuery("DROP TABLE IF EXISTS " + tableName);
+        try {
+            onTrino().executeQuery(format("CREATE TABLE %s AS SELECT date'2019-12-02' c_date", tableName));
+
+            List<Row> expectedStatistics = ImmutableList.of(
+                    isHiveVersionBefore12()
+                            ? row("c_date", null, null, null, null, null, null)
+                            : row("c_date", null, 1.0, 0.0, null, "2019-12-02", "2019-12-02"),
+                    row(null, null, null, null, 1.0, null, null));
+
```

Comment:
Please verify that hive is not broken anymore. 
Trino used to work also before this fix.